superdesk.config.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * This is the default configuration file for the Superdesk application. By default,
+ * the app will use the file with the name "superdesk.config.js" found in the current
+ * working directory, but other files may also be specified using relative paths with
+ * the SUPERDESK_CONFIG environment variable or the grunt --config flag.
+ */
+
+module.exports = function(grunt) {
```

Comment:
not sure if it needs grunt here, can it just return a dict?
imo ideally this file would be loaded into index.html rather then during build, so you can change something there without the need to rebuild, so we can build the app once and use it with different configurations.